files/get-ecr-uri.sh

```diff
@@ -0,0 +1,39 @@
+#!/usr/bin/env bash
+set -euo pipefail
+
```

Comment:
maybe we could put a reference link here as well.
https://docs.aws.amazon.com/eks/latest/userguide/add-ons-images.html